modules/custom/moj_resources/src/TermApiClass.php

```diff
@@ -14,33 +14,6 @@
 
 class TermApiClass
 {
-  /**
-   * Term
-   *
-   * @var array
-   */
-  protected $term;
-  /**
-   * Language Tag
-   *
-   * @var string
-   */
-  protected $lang;
-  /**
-   * Node_storage object
-   *
-   * @var Drupal\Core\Entity\EntityTypeManager
-   */
-  protected $node_storage;
-  /**
-   * Entitity Query object
-   *
-   * @var Drupal\Core\Entity\Query\QueryFactory
-   *
-   * Instance of querfactory
-   */
-  protected $entity_query;
-
```

Comment:
Do we need a field definition for `$this->termStorage`?

Reply:
It's used in `TermApiEndpoint`

Reply:
Resolved